reset-example

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+
+# colors
+RED='\e[0;31m'
+BLUE='\e[0;34m'
+NC='\e[0m' # No Color
+
+
+# warn user that will delete a bunch of files
+printf "This script will ${RED}delete${NC} the your local terraform state, variable files, etc, to "
+printf "reset to example template prior to ${BLUE}./init${NC} and any terraform init/plan/apply you've done.\n"
```

Comment:
This is for state, right? Because running this will not affect any change done in the infra, so we could include something in the text about running `terraform destroy` or dropping manually the stuff created there. Otherwise after reset we'll have errors and conflicts on first tf apply.